walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/StringVerifiedIdSerializer.kt

```diff
@@ -1,18 +1,43 @@
 package com.microsoft.walletlibrary.verifiedid
 
+import com.microsoft.walletlibrary.did.sdk.credential.service.models.contracts.VerifiableCredentialContract
+import kotlinx.serialization.json.Json
+
 object StringVerifiedIdSerializer : VerifiedIdSerializer<String> {
+
+    private val serializer = Json {
+        ignoreUnknownKeys = true
+        isLenient = true
+    }
+
     /**
      * Serialize the given verifiedID into the SerializedFormat
      */
     override fun serialize(verifiedId: VerifiedId): String {
         when (verifiedId) {
             is VerifiableCredential -> {
-                return verifiedId.raw.raw
+                return serializer.encodeToString(
```

Comment:
I'm so sorry since this was a change I asked for; your original change was correct, these serializers have different purposes and if this goes through presentations would fail.

Reply:
Reverted the changes and let there be 2 separate custom serializers for different purposes.